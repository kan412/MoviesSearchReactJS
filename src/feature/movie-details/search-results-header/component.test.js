import React from 'react';
import { shallow } from 'enzyme';
import SearchResultsHeader from './component';

describe('<SearchResultsHeader />', () => {
  it('render without breaking', () => {
    const data = ['action', 'drama'];
    const component = shallow(<SearchResultsHeader genres={data} />);
    expect(component).toMatchSnapshot();
  });
});
