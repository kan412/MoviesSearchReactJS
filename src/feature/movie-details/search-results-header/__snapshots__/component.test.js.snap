// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResultsHeader /> render without breaking 1`] = `
<div
  className="search-results-header"
>
  <div
    className="search-results-header-inner"
  >
    <span>
      Films by  
    </span>
    action
    drama
    <span>
        genre 
    </span>
  </div>
</div>
`;
